src/main/scala/org/broadinstitute/dsde/workbench/sam/service/ManagedGroupService.scala

```
@@ -97,8 +97,10 @@ class ManagedGroupService(private val resourceService: ResourceService, private
       emailLookup <- directoryDAO.batchLoadGroupEmail(ripns.map(ripn => WorkbenchGroupName(ripn.resourceId.value)))
     } yield {
       val emailLookupMap = emailLookup.toMap
-      ripns.map { ripn =>
-        ManagedGroupMembershipEntry(ripn.resourceId, ripn.accessPolicyName, emailLookupMap(WorkbenchGroupName(ripn.resourceId.value)))
+      // This will silently ignore any group where the email could not be loaded. This can happen when a
```

Are we sure we want to "silently ignore"?  Perhaps we should add just a simple log statement to let somebody know that the data is a little fouled up.

---

I don't think this is a big deal nor do I think anything can really be done about it. I think the right thing to do is pretend that it does not exist.